DataFormats/Math/interface/AVXVec.h

```diff
@@ -1,6 +1,8 @@
 #ifndef DataFormat_Math_AVXVec_H
 #define DataFormat_Math_AVXVec_H
 
+#include "DataFormats/Math/interface/SSEVec.h"
```

Comment:
@yamaguchi1024 , is not it going to create a cyclic dependency (SSEVec.h includes AVXVec.h too)?

Reply:
It'll not be a cyclic dependency if headers were in the same module (SSWVec.h and AVXVec.h are both in DataFormatsMath_xr.pcm), but cyclic dependency occurs when headers in the different module are including each other. (Like https://github.com/cms-sw/cmssw/issues/25903#issuecomment-463298741 case)